### PR TITLE
Port remainder of patients + lines scss

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Port the two remaining spare files over.

This involves starting to port over some of the bootstrap config over. To handle stuff that requires reffing bootstrap classes, we throw our bootstrappy config into a single file and import that into the main pack, and ref it in modules as well. (I think webpack is smart enough to just process everything down cleanly.)

This pull request makes the following changes:
* port patient.scss contents into webpack
* port lines.scss contents into webpack
* establish a strategy for reffing webpack components

no view changes

It relates to the following issue #s: 
* Bumps #1854 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

File: app/javascript/styles/lines.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    transform: scale(2);
+    margin: 10px;
+  }
+
+  label {
+    font-size: $font-size-medium;
+  }
+
+  .form-check-label {
+    @extend .ml-8;
```
Comment (colinxfleming) [inline app/javascript/styles/lines.scss:19]: (This ref of the bootstrap utility class is pretty much why we need to ref bootstrap in here.)

File: app/javascript/styles/lines.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  }
+}
```
Comment (colinxfleming) [inline app/javascript/styles/lines.scss:21]: There was an extra button styling here; it wasn't really doing anything so I axed it.

**Review by colinxfleming — COMMENTED** [review]
few notes

**Review by lomky — APPROVED** [review]
❇️

**Pull request merged**